react/helpers/useProgression.jsx

```diff
@@ -0,0 +1,25 @@
+import { useState, useEffect } from 'react'
+
+import isTesting from './isTesting'
+
+const useProgression = () => {
+  const [progression, setProgression] = useState(0)
+
+  const increment = () => {
+    const diff = Math.random() * 10
+    setProgression(progression =>
+      progression >= 100 ? 0 : Math.min(progression + diff, 100)
+    )
+  }
+
+  useEffect(() => {
+    if (!isTesting()) {
+      const interval = setInterval(increment, 500)
+      return () => clearInterval(interval)
+    }
+  }, [])
+
+  return { progression }
+}
```

Comment:
This helper is only for use in test right ? If this is the case, can you put a mention about it in a comment ?

Reply:
Oui c'est le cas, ok pour le commentaire en effet